src/main/java/org/testng/TestNG.java

```diff
@@ -142,14 +143,10 @@
   private final Map<Class<? extends IReporter>, IReporter> m_reporters = Maps.newHashMap();
   private final Map<Class<? extends IDataProviderListener>, IDataProviderListener> m_dataProviderListeners = Maps.newHashMap();
 
-  protected static final int HAS_FAILURE = 1;
-  protected static final int HAS_SKIPPED = 2;
-  protected static final int HAS_FSP = 4;
   protected static final int HAS_NO_TEST = 8;
 
   public static final Integer DEFAULT_VERBOSE = 1;
 
-  private int m_status;
   private boolean m_hasTests= false;
```

Comment:
can be removed and replaced by direct usage of `exitCodeListener`

Reply:
Fixed.